src/main/java/kervyn/Storage.java

```diff
@@ -0,0 +1,127 @@
+package kervyn;
+
+import kervyn.Tasks.Deadline;
+import kervyn.Tasks.Event;
+import kervyn.Tasks.Task;
+import kervyn.Tasks.ToDo;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * The Storage class is responsible for handling file operations such as reading from and writing to the file.
+ */
+public class Storage {
+    private String path;
+    /**
+     * Constructs a Storage object associated with the file path provided.
+     *
+     * @param path The file path where tasks are saved and read from.
+     */
+    public Storage(String path) {
+        this.path = path;
+    }
+
+    /**
+     * Writes the content to the file specified by this.path. If the file or directory does not exist, they will be created.
+     *
+     * @param content The content to be written to the file.
+     * @return Returns 1 if the write operation was successful, 0 otherwise.
+     */
+    public short writeToFile(String content) {
+        try {
+            String[] dirName = this.path.split("/");
+            File dir = new File(dirName[0]);
+
+            if (!dir.exists()) {
+                dir.mkdir();
+            }
+
+            File file = new File(this.path);
+
+            if (!file.exists()) {
+                file.createNewFile();
+            }
+
+            FileWriter fw = new FileWriter(file.getAbsoluteFile(), false);
+//            BufferedWriter bw = new BufferedWriter(fw);
```

Comment:
Comments should be indented relative to their position in the code. Also, it seems like this comment can just be removed.